src/components/shared/form_fields/RadioField.vue

```diff
@@ -16,6 +16,9 @@
 				@update:modelValue="onFieldChange"
 			>
 				{{ option.label }}
+				<div v-if="disabled.includes( option.value )" class="has-text-dark-lighter has-margin-top-18">
```

Comment:
We might ask our UX person what the margin should be, IIR `has-margin-top-18` was part of the removed Bulma helpers